packages/medusa/src/api/routes/admin/notes/index.ts

```diff
@@ -18,3 +19,13 @@ export default (app) => {
 
   return app
 }
+
+export class AdminNoteRes {
+  note: Note
+}
+
```

Comment:
I think we are missing a type `export type AdminNotesListRes` for listing notes.